components/feature/qr/lint.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<lint>
+    <issue id="IconMissingDensityFolder" severity="ignore" />
```

Comment:
I assume this is because of `qr_cam_focus.png`? Something's weird about that. There are x2 and x3 versions but they are all in the `hdpi` folder and not in the correct `*dpi` folder? 🤔

Reply:
heh I missed that when cleaning up. Fixing...

Reply:
Fixed!